docs/wip/theme-shadowing.md

```diff
@@ -1,27 +1,28 @@
+As stated in the [Gatsby documentation](https://www.gatsbyjs.com/docs/how-to/plugins-and-themes/shadowing/): _"This feature allows users to replace a file in the src directory that is included in the webpack bundle with their own implementation."_. Therefore, you have the ability to adapt any file to suit your needs, for example: styles, to match your visual identity, and even change the functionality of a component, to meet your business rules.
 
 ## Shadowing on SFJ
 
 ### What can I shadow?
-In [Gatsby documentation](https://www.gatsbyjs.com/docs/how-to/plugins-and-themes/shadowing/#any-source-file-is-shadowable) you'll learn that you can override(_shadow_) any source file, not only React components. In our example-store implementation we shadowed this typescript [file](https://github.com/vtex/faststore/blob/master/packages/gatsby-plugin-i18n/src/i18n/en.ts) of the gatsby-plugin-i18n to add the expressions of our example-store, turning it into this [file](https://github.com/vtex-sites/storecomponents.store/blob/master/src/%40vtex/gatsby-plugin-i18n/i18n/en.ts). You can customize any file by creating a file in the same path as the theme you are using, example: `.../i18n/en.ts`, that way the Shadowing API will know that it should use your file, instead of the theme's file.
+In the [Gatsby documentation](https://www.gatsbyjs.com/docs/how-to/plugins-and-themes/shadowing/#any-source-file-is-shadowable) you'll learn that you can override (_shadow_) any source file, not just React components. In our example-store implementation, we shadowed this typescript [file](https://github.com/vtex/faststore/blob/master/packages/gatsby-plugin-i18n/src/i18n/en.ts) from `gatsby-plugin-i18n` to add our example-store expressions, transforming it into this [file](https://github.com/vtex-sites/storecomponents.store/blob/master/src/%40vtex/gatsby-plugin-i18n/i18n/en.ts). You can customize any file by creating a file in the same path as the theme you are using, example: `.../i18n/en.ts`, that way the Shadowing API will know that it should use your file, instead of the theme's file.
 
-:warning: Please note that, since the Store Framework Jamstack is built using Gatsby, you are able to use any resource/concept/feature present on Gatsby. Therefore, all types of shadowing explained in the Gatsby documentation works perfectly when you are using the SFJ.
+:warning: Please note that since the Store Framework Jamstack is built using Gatsby, you are able to use any resource/concept/feature present in Gatsby. Therefore, all types of shadowing explained in the Gatsby documentation works perfectly when you are using the SFJ.
```

Comment:
ℹ️ Rember that since Store Framework Jamstack is built using Gatsby, you are able to use any resource/concept/feature present in Gatsby. Therefore, all types of shadowing explained in the Gatsby documentation works perfectly when using the SFJ.